src/app/about-us/members/members.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="row">
+  <div
+    class="col s12 m6 member-card valign-wrapper"
+    *ngFor="let member of members"
+  >
```

Comment:
move this next to the ngFor please

Reply:
Maybe we don't need linebreaks at all... I mean, usually lint rules define a max line length of 140... Formating this with Prettier moves everything to just one line. Sure the linebreaks are for readable reasons, but I think this can be read just fine in one line.